toolbox/index.md

```diff
@@ -79,7 +79,7 @@ Integrated Development Environment (IDE): IntelliJ IDEA
 ![IntelliJ IDEA](intellij-idea.png){:width="344"}
 </figure>
 
-**What it is.** An IDE allows you to edit source code, invoke the compiler, interact with the debugger, and so forth.
```

Comment:
I don’t like this change, because it turns the heading into a question. In fact, I’d argue that the modified version would deserve a question mark: “What is it?” I believe assertions are better than questions—but I accept it’s a matter of style. In any case, if we were to carry out this change, I think we should change the other headings similarly, for example, “How to Learn it.” would turn into “How do I learn it?”

What do you think?

Reply:
About your argument here, 'Where to Get It' is also a question, rather than an assertion, and it is not succeeded by a question mark. Should we just stick with 'What is it', in the same spirit?

Reply:
I don’t read “where to get it” as a question. Try starting with “this is …” for example, “this is **what it is**” and “this is **where to get it**.”

Reply:
I don't think that is what it comes off like. I didn't attach the 'this is' prefix before you told me to.

Reply:
How about we turn them all into questions?

| This | turns into that |
|-|-|
| What it is. | What Is It? |
| Our Choice. | Why Did We Choose It? |
| Where to Get it. | Where Do I Get It? |
| How to Learn it. | How Do I Learn It? |

Doesn’t get much more explicit than that 😛

Please remember to change **§ How to Read** accordingly.